oidc-api/src/main/java/uk/gov/di/authentication/oidc/entity/AuthUserInfoClaims.java

```diff
@@ -15,7 +15,8 @@ public enum AuthUserInfoClaims {
     PHONE_VERIFIED("phone_number_verified"),
     SALT("salt"),
     VERIFIED_MFA_METHOD_TYPE("verified_mfa_method_type"),
-    CURRENT_CREDENTIAL_STRENGTH("current_credential_strength");
+    CURRENT_CREDENTIAL_STRENGTH("current_credential_strength"),
+    NEW_ACCOUNT("new_account");
```

Comment:
Worth setting this on the auth-external-api version as well?

Also looks like there are a couple of other usages of `"new_account"` that could be replaced